addons/website_hr_recruitment/controllers/main.py

```diff
@@ -3,45 +3,55 @@
 
 from odoo import http, _
 from odoo.addons.http_routing.models.ir_http import slug
+from odoo.osv.expression import AND
```

Comment:
Commit title: `[IMP] website_hr_recruitment: add pager and search` ?

Reply:
Done 👍